index.html

```diff
@@ -30,7 +30,10 @@
                 <button class="btn" onclick="due()">2</button>
                 <button class="btn" onclick="tre()">3</button>
                 <button class="btn" onclick="quattro()">4</button>
-                <button class="btn" onclick="cinque()">5</button>
+                <button class="btn" onclick="cinque()">5</button><br>
+                <h3>Options</h3>
```

Comment:
I don't like very much this section. We could think about an input element with the replaceStr value, so the user can try with another string. But I would like to do this in an other PR because I'd like to close this and keep it clean.

What I'm doubtful about is the second reset button: they are identical and it's not clear the meaning of that option section.

We can further do this adding the <input> I was talking about and with some text that references to the replaceStr meaning. 

So, I would not change the demo in this moment.

Reply:
That makes sense. I have now reverted the changes to the demo page and updated the branch.